lib/smooth/include/smooth/application/io/i2c/DS3231.h

```diff
@@ -0,0 +1,351 @@
+// Smooth - C++ framework for writing applications based on Espressif's ESP-IDF.
+
```

Comment:
What do I need to look for to make sure this is in UTF8?
I thought it would be by default. Is it possible that my Git for Windows is messing with file formats?

Reply:
I don't know how it looks in VS, but in VS Code, there's a text you can click in in the lower right corner:
![image](https://user-images.githubusercontent.com/2729388/64555881-94d60d80-d33e-11e9-804e-bd381fa063c2.png)